### DISCUSSION TITLE
How do I use the AZ power shell modules

### DISCUSSION THREAD

**Discussion by Dirtbagg** [actions/virtual-environments#7252] [General questions]
I'm looking to using the Power Shell Modules Az: 9.3.0. I can't find any documentation on it.
Also is there a way to run the AzureAD module on a hosted windows server?
Thanks

**Comment by mikhailkoliada** [comment]
Hello!

More info could be found on readme here: https://github.com/Azure/az-ps-module-versions

**Reply by Dirtbagg** [reply]
OK, one more question, how would I be able to use AzureAd cmdlets like New-AzureADApplication?
